vavr/src/main/java/io/vavr/Lazy.java

```diff
@@ -101,7 +101,7 @@ private Lazy(Supplier<? extends T> supplier) {
      */
     public static <T> Lazy<Seq<T>> sequence(Iterable<? extends Lazy<? extends T>> values) {
         Objects.requireNonNull(values, "values is null");
-        return Lazy.of(() -> Vector.ofAll(values).map(Lazy::get));
+        return Lazy.of(() -> Vector.ofAll(values).map(Lazy<? extends T>::get));
```

Comment:
i think this is a bug in both the JDK and in Idea, they report different things for every variation of the code

Reply:
An alternative suggestion that Idea accepts also:
```java
@SuppressWarnings("Convert2MethodRef") // TODO fixme on Java 9
public static <T> Lazy<Seq<T>> sequence(Iterable<? extends Lazy<? extends T>> values) {
    Objects.requireNonNull(values, "values is null");
    return Lazy.of(() -> Vector.ofAll(values).map(v -> v.get()));
}
```